### PR TITLE
Add object type information for ncbi taxon->object query

### PR DESCRIPTION
**Pull request by jayrbolton**
* Add a few useful fields for type vertices in the `ws` namespace
* Return the object type in the ncbi_tax_get_associated_ws_objects query

### CODE REVIEW

File: stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ params:
       items: {type: string}
       description: Taxon edge fields to keep in the results
       default: null
-query_prefix: WITH ws_object_version
+query_prefix: WITH ws_object_version, ws_type_version
 query: |
   let count = COUNT(
```
Comment (MrCreosote) [inline stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml]: this seems to be returning a count of all object versions attached to the taxon, including objects to which the user has no access. That seems wrong, naively.

File: stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml [hunk 2]
```diff
@@ -48,12 +48,15 @@ query: |
...
-      for obj, e in 1..1 inbound tax ws_obj_version_has_taxon
-        filter obj.is_public or obj.workspace_id IN ws_ids
-        limit @offset, @limit
-        return {
-          ws_obj: @select_obj ? KEEP(obj, @select_obj) : obj,
-          edge: @select_edge ? KEEP(e, @select_edge) : e
-        }
+      for obj, e in 1 inbound tax ws_obj_version_has_taxon
+        for type in 1 outbound obj ws_obj_instance_of_type
+          filter obj.is_public or obj.workspace_id IN ws_ids
```
Comment (MrCreosote) [inline stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml]: Should this filter be in the outer loop?

File: test/stored_queries/test_ncbi_tax.py [hunk 3]
```diff
@@ -285,3 +296,4 @@ def test_get_associated_objs(self):
         ids = {ret['ws_obj']['_id'] for ret in results['results']}
         self.assertEqual(assignments, {'assn1', 'assn2'})
         self.assertEqual(ids, {'ws_object_version/1:1:1', 'ws_object_version/1:1:2'})
+        self.assertEqual(results['results'][0]['ws_obj']['type']['type_name'], 'Genome')
```
Comment (MrCreosote) [inline test/stored_queries/test_ncbi_tax.py]: I'd probably check the entire type structure

File: stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml (hunk 1 above)
Comment (jayrbolton) [reply stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml]: This is what the frontend/product folks requested. A little bit of a security leak, but I think negligible :man_shrugging:

File: stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml (hunk 2 above)
Comment (jayrbolton) [reply stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml]: Good call, fixed

File: test/stored_queries/test_ncbi_tax.py (hunk 3 above)
Comment (jayrbolton) [reply test/stored_queries/test_ncbi_tax.py]: Good call, fixed

**Pull request merged**